core/src/main/java/com/ultramixer/jarbundler/PropertyListWriter.java

```diff
@@ -281,90 +270,141 @@ private void buildDOM()
             writeDocumentTypes(documentTypes, dict);
         }
 
-        // Java / JavaX entry in the plist dictionary
-        writeKey(bundleProperties.getJavaXKey() ? "JavaX" : "Java", dict);
-        Node javaDict = createNode("dict", dict);
+        // Java / JavaX entries in the plist dictionary
+        if (bundleProperties.getJavaVersion() < 1.7)
```

Comment:
`JavaX` key must stay supported for latest Java versions as well. Blocker for this PR.

Reply:
Didn't realise. Sorry.